File: code-coaches/easy/hovercraft/Hovercraft.swift
func hovercraftProfitStatus(unitsSold: Int) -> String {
    let sellPrice = 3_000_000
	let costPrice = 2_000_000
	let insurancePrice = 1_000_000

	let unitsBuilt = 10
	let expenses = (unitsBuilt * costPrice) + insurancePrice
	let revenue = unitsSold * sellPrice

	if revenue > expenses {
		return "Profit"
	} else if revenue < expenses {
		return "Loss"
	} else {
		return "Broke Even"
	}
}

func main() {
    let unitsSold = Int(readLine()!)!
	print(hovercraftProfitStatus(unitsSold: unitsSold))
}

main()

File: code-coaches/easy/hovercraft/Hovercraft.cs
using System;

public class Hovercraft
{
    static string hHvercraftProfitStatus(int unitsSold)
    {
	    int sellPrice = 3000000;
		int costPrice = 2000000;
		int insurancePrice = 1000000;

		int unitsBuilt = 10;
		int expenses = (unitsBuilt * costPrice) + insurancePrice;
		int revenue = unitsSold * sellPrice;

		if (revenue > expenses)
			return "Profit";
		else if (revenue < expenses)
			return "Loss";
		else
			return "Broke Even";
	}

    static void Main(string[] args)
    {
        int unitsSold = int.Parse(Console.ReadLine());
        Console.WriteLine(hHvercraftProfitStatus(unitsSold));
    }
}
